core/commands/storage/path/path.go

```diff
@@ -264,6 +266,41 @@ var PathMigrateCmd = &cmds.Command{
 	},
 }
 
+var PathListCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "list directories",
+		ShortDescription: "list directories",
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("parent", true, true,
+			"parent path, should be absolute path."),
+	},
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		list, err := list(req.Arguments[0])
+		if err != nil {
+			return err
+		}
+		return cmds.EmitOnce(res, list)
+	},
+	Type: []string{},
```

Comment:
There is a `stringList` type. It's better wrapping it inside a struct so it can be extended with attributes if needed.